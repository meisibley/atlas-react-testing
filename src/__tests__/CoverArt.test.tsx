import React from 'react';
import { render } from '@testing-library/react';
import { expect, test } from "vitest";
import CoverArt from '../components/CoverArt';

test('renders correctly', () => {
    const coverImage = 'https://raw.githubusercontent.com/atlas-jswank/atlas-music-player-api/main/images/albumn4.webp';
    const { container } = render(<CoverArt cover={coverImage} />);
    
    expect(container).toMatchSnapshot();
});
