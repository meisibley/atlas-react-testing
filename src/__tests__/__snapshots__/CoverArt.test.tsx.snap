// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders correctly 1`] = `
<div>
  <div>
    <img
      alt="Cover Art"
      class="rounded-md"
      src="https://raw.githubusercontent.com/atlas-jswank/atlas-music-player-api/main/images/albumn4.webp"
    />
  </div>
</div>
`;
